### ISSUE TITLE
VBA QA: Accordions on VBA Prepare for your visit should be conditional

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17852]
## Description
On VBA, the "Expand all" button currently display when there are no accordions because there is no conditional written. While there is content that will be required for this section so there should be no VBA pages published without Prepare for your visit content, we should make sure this is conditionalized properly.

![Screenshot 2024-04-11 at 10 35 15 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0c44f4d7-90ee-4600-aac1-be3dcfb5be97)

## Eng notes
1. This field is not mechanically required in Drupal, but VBA facilities won't be approved for launch / publish without this data present
2. If the accordion is single select (only 1 accordion can open at a time), Expand All does not appear. But any other setting, Expand All always appears. Laura will raise with the DS.

This AC is blocked by DS component fix for 2 above:
- [ ] "Expand all" should only show if more than 1 accordion item is populated

## Acceptance Criteria
- [ ] Expand All, Heading, and content should only show if accordion group is populated
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by jilladams** [comment]
Not launch blocking, but should do before we walk away from MVP.

**Comment by Agile6MSkinner** [comment]
@laflannery The description says that you would raise a blocker with DS. What was the outcome of that?

**Comment by laflannery** [comment]
I haven't done this yet - it's low priority but also isn't a blocker for this ticket. The 3 current AC should be able to to be completed without any additional functionality needed from the component

**Comment by Agile6MSkinner** [comment]
Technically there is an edge-case risk, but there are multiple systems in place that would prevent this from coming up. Will close as not planned.
